### DISCUSSION TITLE
Sudden error of "No exports found in module "../../index.js"."

### DISCUSSION THREAD

**Discussion by jayesh-vyas** [vercel/community#5444] [Help]
### Summary

No exports found in module "../../index.js".
Did you forget to export a function or a server?
INIT_REPORT Init Duration: 5753.24 ms	Phase: init	Status: error	Error Type: Runtime.ExitError
No exports found in module "../../index.js".
Did you forget to export a function or a server?
INIT_REPORT Init Duration: 6306.13 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError
Error: Runtime exited with error: exit status 1
Runtime.ExitError

this error has been coming suddenly
my deployment was running fine 2 days later and then suddenly this started appearing and now it doesnt work
i have only one index.js file everything was working fine 2 days ago

### Steps to Reproduce

No exports found in module "../../index.js".
Did you forget to export a function or a server?
INIT_REPORT Init Duration: 5753.24 ms	Phase: init	Status: error	Error Type: Runtime.ExitError
No exports found in module "../../index.js".
Did you forget to export a function or a server?
INIT_REPORT Init Duration: 6306.13 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError
Error: Runtime exited with error: exit status 1
Runtime.ExitError

**Comment by ducsilva** [comment]
i have the same trouble. Any fix it's?

**Comment by sahilatahar** [comment]
If your deployment was working fine but now shows "No exports found in module", try these steps:

1. **Check Exports in `index.js`**:
   ```javascript
   // For Express.js
   const express = require('express');
   const app = express();
   
   module.exports = app;
   ```

2. **Review Recent Changes**: Revert any recent changes that might have caused the issue.

3. **Reinstall Dependencies**:
   ```bash
   rm -rf node_modules package-lock.json
   npm install
   ```

4. **Re-deploy**: Sometimes, re-deployment fixes transient issues.

5. **Check Logs**: Look at deployment logs for more details.
